ui/app/templates/components/identity/item-aliases.hbs

```diff
@@ -13,9 +13,14 @@
             glyph='role'
             size=14
             class="has-text-grey-light"
-            }}{{item.name}}</a>
-        <span class="tag">{{item.mountType}}</span>
-        <code class="has-text-grey is-size-8">{{item.mountAccessor}}</code>
+            }}<span class="has-text-weight-semibold">{{item.name}}</span></a>
+        <div class="has-text-grey">
+          {{item.id}}
+        </div>
+        <span class="tag"> {{item.mountType}} </span>
```

Comment:
Did this need spaces between for a reason?

Reply:
Lol nope, was being hasty - will fix!